src/runtimes/node/parser/exports.ts

```diff
@@ -187,13 +224,20 @@ const parsePrimitive = (exp: Expression | PatternLike): PrimitiveResult => {
   }
 }
 
-// Tries to resolve the export from a binding (variable)
-// for example `let handler; handler = () => {}; export { handler }` would
-// resolve correctly to the handler function
-const getExportsFromBindings = (specifiers: ExportNamedDeclaration['specifiers'], getAllBindings: BindingMethod) => {
-  const specifier = specifiers.find(isHandlerExport)
-
-  if (!specifier) {
+/**
+ * Tries to resolve the export with a given name from a binding (variable).
+ * For example, the following would resolve correctly to the handler function:
+ *
+ * `let handler; handler = () => {}; export { handler }`
+ */
+const getExportsFromBindings = (
+  specifiers: ExportNamedDeclaration['specifiers'],
+  name: string,
+  getAllBindings: BindingMethod,
+) => {
+  const specifier = specifiers.find((node) => isNamedExport(node, name))
```

Comment:
here you get only the named exports what about doing an object literal export over a default?

```javascript
const handler = () => {}

export default {
  handler
}
```

Reply:
same would apply to the cjs where it's:

```javascript
const handler = () => {}

exports.default {
  handler
}
```